examples/dreambooth/train_dreambooth.py

```diff
@@ -213,6 +213,11 @@ def parse_args(input_args=None):
             raise ValueError("You must specify a data directory for class images.")
         if args.class_prompt is None:
             raise ValueError("You must specify prompt for class images.")
+    else:
+        if args.class_data_dir is not None:
+            raise ValueError("You need not specify a data directory for class images.")
+        if args.class_prompt is not None:
+            raise ValueError("You need not specify prompt for class images.")
```

Comment:
Here we should check for `instance_data_dir` and `instance_prompt`. Also `instance_data_dir` is already a required argument. So instead of this checks could you make `instance_prompt` a required arg ? 

If both of those are required that script won't run if they are missing.

Reply:
Nice suggestion!
I updated it. (f89839b)
I also noticed a needless check, so I removed it. (df83d4d)